beta/requirements.txt

```diff
@@ -7,6 +7,7 @@ tensorflow_addons==0.12.0
 tensorflow_hub
 texttable
 opencv-python
+pyyaml
```

Comment:
It looks like a new dependency for the NNCF. @vshampor  Is it an approved dependency?

Reply:
It is not, and in fact it should not be used for security reasons. YAML parsing in general is insecure, so we should avoid having to handle .yml files ourselves.

Reply:
This package is used to modify AC configs.
Now we have configs in beta/tests/tensorflow/data/ac_configs/ only for FP32 baselines models, and in tests we modify them on-the-fly for compressed models, this modification was discussed with developers by @daniil-lyakhov

Reply:
This requirement is not for the NNCF package then, but for the TF tests. Have the requirements.txt been separated for TF yet?

Reply:
And in fact it used other OpenVino frameworks and tools, like training_extensions and Accuracy Checker.
I can separate req's and add requirements.txt in tests/ or we need add full configs for all models.

Reply:
Updated